packages/themes/scss/sn.scss

```diff
@@ -0,0 +1,17 @@
+
```

Comment:
we should rename this theme to something else other than sn

Reply:
resolved! renamed it to orange-and-black.